nuxeo-document-preview.html

```diff
@@ -253,13 +253,11 @@
             this.xpath === 'file:content'
           ) {
             return this.document.properties['vid:transcodedVideos']
-              .filter(
-                (conversion) =>
-                  conversion &&
+              .filter((conversion) =>
+                conversion &&
                   conversion.content &&
                   conversion.content.data &&
-                  conversion.content['mime-type'].match(/^video.*/),
-              )
+                  conversion.content['mime-type'].match(/^video.*/))
```

Comment:
An indentation level was removed above, we can have all conditions on the same level:
```
conversion &&
conversion.content &&
conversion.content.data &&
conversion.content['mime-type'].match(/^video.*/))
```

Reply:
We can, but it wasn't me changing it. I used `eslint` with `--fix` option to solve the linting issues 🤷

Reply:
Done 👍 Thanks